### PR TITLE
DYN-4997 Add Trust Location Added/Skipped Notification

### PR DESCRIPTION
**Pull request by QilongTang**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

Per https://jira.autodesk.com/browse/DYN-4997, add trust location added/ skipped notification

Trust location added with trust warning check box:
image

When user click No and close the file
image

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

Add trust location added/ skipped notification

### Reviewers

@DynamoDS/dynamo 

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/DynamoCoreWpf/Views/FileTrust/FileTrustWarning.xaml.cs [hunk 1]
```diff
@@ -111,7 +111,10 @@ private void CloseFileButton_Click(object sender, RoutedEventArgs e)
         {
             fileTrustWarningViewModel.ShowWarningPopup = false;
             if (dynViewModel.CloseHomeWorkspaceCommand.CanExecute(null))
+            {
                 dynViewModel.CloseHomeWorkspaceCommand.Execute(null);
+                dynViewModel.MainGuideManager.CreateRealTimeInfoWindow(string.Format(Properties.Resources.TrustLocationSkippedNotification, fileTrustWarningViewModel.DynFileDirectoryName));
```
Comment (zeusongit) [inline src/DynamoCoreWpf/Views/FileTrust/FileTrustWarning.xaml.cs:116]: Can we add a todo, or file a task to make this API generic?

**Comment by zeusongit** [comment]
Does this banner also has the bug, where it is not bound to the parent window?
DynamoSandbox_VrSviTOFhw

**Comment by QilongTang** [comment]
> Does this banner also has the bug, where it is not bound to the parent window? DynamoSandbox_VrSviTOFhw [ ](https://user-images.githubusercontent.com/32665108/172312573-3469ec5e-ad4e-44b0-bed2-d064648f6fdd.gif) DynamoSandbox_VrSviTOFhw [ ](https://user-images.githubusercontent.com/32665108/172312573-3469ec5e-ad4e-44b0-bed2-d064648f6fdd.gif)

Yes, a lot of Dynamo UI components come with such bug

**Review by zeusongit — APPROVED** [review]

**Comment by QilongTang** [comment]
Added one small fix to reset run status message in Manual mode.

**Comment by QilongTang** [comment]
Updated the messages so graph in Manual mode is expected to not run. FYI: @Amoursol 
image

**Comment by QilongTang** [comment]
No real regressions on job https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/6338/

**Comment by QilongTang** [comment]
@zeusongit @reddyashish This PR is ready for a final look

**Review by reddyashish — APPROVED** [review]

**Pull request merged**